src/NFun.Fuspec.ConsoleTestHandler/Program.cs

```diff
@@ -1,12 +1,60 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Nfun.Fuspec.Parser;
+using Nfun.Fuspec.Parser.Model;
+using NFun.Fuspec.TestHandler;
+using NFun.Fuspec.TestHandler.Models;
 
 namespace NFun.Fuspec.ConsoleTestHandler
 {
     class Program
     {
         static void Main(string[] args)
         {
+            var ff = new TryFindFuspecTests();
+            ff.FuspecTests();
+            Console.ReadLine();
             Console.WriteLine("Hello World!");
         }
     }
+
+    public class TryFindFuspecTests
+    {
+        private const string FuspecTestExtension = "*.fuspec";
+
+        public void FuspecTests()
```

Comment:
[Naming] Method should be verb. For ex "ReadFromDirectory"

Reply:
Class contains no state. It has only one method. Actually it is procedure - so make the class and the method - static